torch/utils/_cxx_pytree.py

```diff
@@ -20,6 +20,19 @@
 from typing_extensions import deprecated, TypeIs
 
 import optree
+
+from torch._vendor.packaging.version import Version
```

Comment:
This makes me sad, as there is torch.Version, that supposed to do the same, isn't there...